### DISCUSSION TITLE
how to use it on one computer without a data center and a cluster?

### DISCUSSION THREAD

**Discussion by widg** [vmware/packer-examples-for-vsphere#27] [Help] [answered]
how to use it on one computer without a data center and a cluster? if you leave the values of the variables empty, then an error.

```
CONFIRM: Build a Ubuntu Server 20.04 LTS Template for VMware vSphere.

Continue? (y/n)
y
Building a Ubuntu Server 20.04 LTS Template for VMware vSphere ...
Initializing Hashicorp Packer and required plugins ...
Applying the HashiCorp Packer Build ...
Error: 1 error(s) occurred:

* 'host' or 'cluster' is required

  on ubuntu-server-20-04-lts.pkr.hcl line 345:
  (source code not available)



==> Wait completed after 6 microseconds

==> Builds finished but no artifacts were created.
Done.
```

**Comment by tenthirtyam** [comment, marked as answer]
Hi! This project is designed to provide working examples for using Packer with a vSphere environment that includes a vCenter Server instance. It uses the objects, such as, datacenter, cluster, and content library.

For environments without a vCenter Server, you will need to modify the project examples to use `host` instead of the `vcenter_server` and remove the `datacenter`, `cluster`, `folder` and `content_library` constructs.

Thanks,
Ryan
